net/wireguard/src/www/widgets/widgets/wireguard.widget.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * Copyright (C) 2020 Deciso B.V.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in the
+ *    documentation and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+ * INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY
+ * AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ * AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY,
+ * OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+require_once("guiconfig.inc");
+require_once("widgets/include/wireguard.inc");
+
+$data = configd_run("wireguard widget");
+
+$empty = strlen($data) <= 2;
+
+?>
+
+<?php if (!$empty):
+    $handshakes = explode(";", $data); ?>
+
+<table class="table table-striped table-condensed">
+    <thead>
+        <tr>
+            <th><?= gettext("Interface") ?></th>
+            <th><?= gettext("Endpoint") ?></th>
+            <th><?= gettext("Latest Handshake") ?></th>
+        </tr>
+    </thead>
+    <tbody>
```

Comment:
in general it is preferable to always print the header and use colspan="3" to print a message for no instances with a single td

Reply:
I made the change but the style will be different (in comparison to the OpenVPN widget).

![comparison](https://user-images.githubusercontent.com/18613935/84367524-a5b8f580-abd4-11ea-8437-e4cff307ddd1.png)

Reply:
Or did you mean adding a `td colspan="3"` in the `tbody` section for `no instances`?